letsencrypt/models/letsencrypt.py

```diff
@@ -2,73 +2,70 @@
 # © 2016 Therp BV <http://therp.nl>
 # © 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-import os
+from os.path import join, isdir, isfile
+from os import makedirs
+from odoo import _, api, models, exceptions
+from odoo.tools import config
 import logging
-import urllib2
 import urlparse
 import subprocess
-import tempfile
-from odoo import _, api, models, exceptions
-from odoo.tools import config
+import requests
+import base64
+_logger = logging.getLogger(__name__)
+
+try:
+    from cryptography.hazmat.backends import default_backend
+    import josepy as jose
+    from cryptography.hazmat.primitives.asymmetric import rsa
+    from cryptography.hazmat.primitives import serialization
+    from acme import client, crypto_util
+    from acme.messages import NewRegistration
+    import IPy
+except ImportError as e:
+    _logger.error(e)
```

Comment:
make this a debug message, as addons are loaded python wise even when they aren't installed